src/@carbon-info/views/Ecosystem/Ecosystem.tsx

```diff
@@ -64,18 +65,100 @@ const Ecosystem: React.FC = () => {
     <Box className={classes.filters}>
-      {filters.map((filter, index) => {
+      {dAppsFilters.map((filter, index) => {
+        return (
+          <Box className={classes.filterBox} key={`${filter}-${index}`}>
+            <Button className={clsx(classes.filterButton, { active: dAppsFilter === filter })} onClick={() => setDAppsFilter(filter)} disableRipple>
+              {filter}
+            </Button>
+          </Box>
+        );
+      })}
+    </Box>
+  );
+
+
+
+  const filteredDApps = React.useMemo(() => {
+    let filtered: DAppsConfig[] = [];
+    switch (dAppsFilter) {
+      case "Carbon Core":
+        filtered = allDApps.filter((blockchain) => {
+          return blockchain.category === "Carbon Core";
+        });
+        break;
+      case "Carbon EVM":
+        filtered = allDApps.filter((blockchain) => {
+          return blockchain.category === "Carbon EVM";
+        });
+        break;
+      case "All":
+        filtered = allDApps;
+        break;
+      default:
+        filtered = allDApps;
+        break;
+    }
+    return filtered;
+  }, [allDApps, dAppsFilter]);
+
+  const renderSubtext = React.useMemo(() => {
+    let subtext;
+    switch (dAppsFilter) {
+      case "Carbon EVM":
+        subtext = (
+          <>
+            <Typography variant="body1" color="textSecondary" align="left" style={{ maxWidth: "1024px", marginTop: "20px" }}>
+              The EVM component of Carbon allows anyone to deploy EVM-based smart contracts written in Solidity, Vyper, etc. on Carbon. This component allows users to perform any action via both Cosmos and EVM formatted transactions, meaning that users and developers can use popular Ethereum wallets and clients (e.g. MetaMask, HardHat, etc.) to interact with Carbon without additional effort.
+            </Typography>
+            <CTAButton
+              text="Learn More"
+              link={Path.Docs.CarbonEVM}
+              textClassName={classes.ctaButtonText}
+              iconClassName={classes.ctaButtonIcon}
+            />
+          </>
+        );
+        break;
+      case "Carbon Core":
+        subtext = (
+          <>
+            <Typography variant="body1" color="textSecondary" align="left" style={{ maxWidth: "1024px", marginTop: "20px" }}>
+              Carbon Core consists of various native modules written in native code (e.g. Golang) instead of a virtual machine. This implementation securely enables features in a scalable manner such as on-chain central-limit order books, lending and borrowing markets, and more.
+            </Typography>
+            <CTAButton
+              text="Learn More"
+              link={Path.Docs.CarbonCore}
+              textClassName={classes.ctaButtonText}
+              iconClassName={classes.ctaButtonIcon}
+            />
+          </>
+        );
+        break;
+      default:
+        subtext = null;
+    }
+    return subtext;
+  }, [allDApps, dAppsFilter]);
+
+
+  const blockchainFilterButtons = (
```

Comment:
Notice this is rendered separately from the return statement. Think you can just plonk the render code inside the return statement instead.
i.e.
```typescript
<>
  {value === "Blockchains" && (
    <>
      <Box className={classes.filters}>
        {blockchainFilters.map((filter, index) => { ... })}
        ...
      </Box>
    </>
  )}
</>
```